actors/builtin/miner/policy.go

```diff
@@ -47,3 +47,9 @@ func precommitDeposit(sectorSize abi.SectorSize, duration abi.ChainEpoch) abi.To
 func temporaryFaultFee(weights []*power.SectorStorageWeightDesc, duration abi.ChainEpoch) abi.TokenAmount {
 	return big.Zero() // PARAM_FINISH
 }
+
+// MaxFaultsCount is the maximum number of faults that can be declared
+const MaxFaultsCount = 32 << 20
+
+// ProvingPeriod defines the frequency of PoSt challenges that a miner will have to respond to
+const ProvingPeriod = 300
```

Comment:
That's really quick!

It occurs to me this might be one of those things where we want to inject values here at build time for testing networks.

Reply:
Yeah, thats why i was wondering where it should live. In lotus we have this defined in a global 'build' package that has different values if we're in tests vs debug mode vs testnet mode

Reply:
As a first step, I would be ok with moving these out to a structure and then having two versions of that structure, and a static to chose between them (not this PR)? It's a bit inflexible. There must be better ideas.